program-workflows/hiv-recency-testing.md

```diff
@@ -56,17 +56,54 @@ This data is entered following the steps in [HIV Testing Services Client Card](.
 ## Reporting
 
 The reporting tools include the national HMIS tools are:  
-1. HMIS ACP 019 HIV Testing Services Register  
+1. HMIS ACP 018 HIV Testing Services Register  
 2. HMIS 105 Health Unit Outpatient Monthly Report - Section 4  
 3. MER Indicators - HTS\_RECENT, HCT\_TST\_Facility  
 4. Data Exports
 
 * HTS Client Card Export - CSV file containing all the fields entered on the HTS Card 
 * Recency Data Export - CSV file with recency data fields 
 
+
+
 ## Data Sharing with Central Server
 
-This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds
+This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds. The EMR gives the option to change the scheduled sync frequency.
+
+### Sync data flow
+
+![](/assets/sync-data-flow.png)
```

Comment:
@mugumedavid add ALT Text (in the square brackets) and use a relative path to the image this one is broken